Failed to add calender entry (got an unexpected keyword argument 'path')

When I try to add calendar entries from Thunderbird, I get the following message (and the entry will not become visible on other devices)

```
Mai 16 09:41:11 radicale[380420]: [380420/Thread-1440 (process_request_thread)] [INFO] Successful login: '<REDACTED>' (htpasswd)
Mai 16 09:41:11 radicale[380420]: [380420/Thread-1440 (process_request_thread)] [INFO] PUT request for '/<REDACTED>/Z3bps4UpfB8nCtcnvUgKNt1-9g0L2R4s/5482d0ed-464d-4d01-aadc-390558b171d7.ics' received from 127.0.0.1 using 'Mozilla/5.0 (X11; Linux x86_64; rv:138.0) Gecko/20100101 Thunderbird/138.0'
Mai 16 09:41:11 uwsm_sway.desktop[3737079]: console.warn: Calendar: "[calCachedCalendar] Unable to perform playback action add to the server, will try again next time (f4383b58-2b7f-4c4b-af70-edc8c3278ad4,Server Replied with 500)"
Mai 16 09:41:11 uwsm_sway.desktop[3737079]: console.error: Calendar: "[calCachedCalendar] Could not perform playback operation add for item <REDACTED>: Server Replied with 500"
Mai 16 09:41:11 radicale[380420]: [380420/Thread-1439 (process_request_thread)] [ERROR] An exception occurred during PUT request on '/<REDACTED>/Z3bps4UpfB8nCtcnvUgKNt1-9g0L2R4s/f4383b58-2b7f-4c4b-af70-edc8c3278ad4.ics': Storage.acquire_lock() got an unexpected keyword argument 'path'
```

--- a/etesync_dav/radicale/storage.py
+++ b/etesync_dav/radicale/storage.py
@@ -605,7 +605,7 @@ class Storage(BaseStorage):
         raise NotImplementedError
 
     @contextmanager
-    def acquire_lock(self, mode, user=None):
+    def acquire_lock(self, mode, user=None, *args, **kwargs):
         """Set a context manager to lock the whole storage.
 
         ``mode`` must either be "r" for shared access or "w" for exclusive

